app-editors/enter-tex/enter-tex-3.47.0.ebuild

```diff
@@ -0,0 +1,74 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit gnome.org gnome2-utils meson vala xdg
+
+DESCRIPTION="Integrated LaTeX environment for GNOME"
+HOMEPAGE="https://gitlab.gnome.org/swilmet/enter-tex"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~riscv ~x86"
+IUSE="gtk-doc test"
+RESTRICT="!test? ( test )"
+
+DEPEND="
+	>=dev-libs/glib-2.80:2
+	>=x11-libs/gtk+-3.22:3
+	>=app-text/gspell-1.8:0=
+	>=gui-libs/libgedit-tepl-6.11:=
+	>=dev-libs/libgee-0.10:0.8=
+	gnome-base/gsettings-desktop-schemas
+	x11-libs/gdk-pixbuf:2
+	x11-libs/pango
+	gnome-base/dconf
+"
+#	introspection? ( >=dev-libs/gobject-introspection-1.30.0:= )
```

Comment:
The build is now using vala for the most port. I think  g-i is still needed to build Gtex-1.gir. Please advise as I haven't done this kind of things in a long time.